index.html

```diff
@@ -155,25 +155,30 @@ <h3 class="mk-heading mk-heading--md"><a href="{{ site.baseurl }}/community-reso
     </div>
 
   </section>
-<section class="mk-faqs mk-main__section">
+<!-- <section class="mk-faqs mk-main__section">
```

Comment:
let's remove this section instead of just commenting it out

Reply:
Done👍🏽